packages/terra-notification-dialog/src/NotificationDialog.jsx

```diff
@@ -186,11 +215,23 @@ class NotificationDialog extends React.Component {
                 <div className={cx('text-wrapper')}>
                   {title
                     && <div id="notification-dialog-title" className={cx('title')}>{title}</div>}
-                  {message
-                    && <div className={cx('message')}>{message}</div>}
+                  {(startMessage || content || endMessage)
+                    && (
+                      <div className={cx('message')}>
```

Comment:
Do any of styles from the message class need to be put on the individual divs below? Are there any problems with wrapping the `content` with this styled div?

Reply:
The only ones that probably need the message class are the `startMessage` and `endMessage`. `content` probably doesn't need the class applied to it. 

With this in mind, I changed it to be 3 separate `<div>`s added on their own with only `startMessage` and `endMessage` getting the `message` class here https://github.com/cerner/terra-framework/pull/899/commits/9fb621a134f79a08d05db32ebf10603b3027fc3a#diff-cb62dc7ce0197bc569a9952450bb9fe5R218